packages/backend-core/src/objectStore/utils.ts

```diff
@@ -29,3 +31,27 @@ try {
 export function budibaseTempDir() {
   return bbTmp
 }
+
+export const bucketTTLConfig = (
+  bucketName: string,
+  days: number
+): PutBucketLifecycleConfigurationRequest => {
+  const lifecycleRule = {
+    ID: `${bucketName}-ExpireAfterOneDay`,
```

Comment:
We should either do the id dynamic, or remove the days parameters. Otherwise we will have `ExpireAfterOneDay` rules that expire after 6 `days`